get_forecast.py

```diff
@@ -342,7 +342,18 @@ def format_rows(output,account_width):
     }
     
     output_rows.append(row_headings)
+    
+    # add a separator row
+    separator_row = {
+        "Account": "".ljust(account_width, "-"),
+        "MTD": "".ljust(mtd_width, "-"),
```

Comment:
does this work correctly if we change the configuration of the columns.   in the config we can remove columns and re-arrange them.  did you test with and without MTD and re-arrange the columns in the config variable?

Reply:
Yes. I tried with and without MTD, and it works perfectly.

Reply:
I also tried with only MTD, and removing Forecast. Still works fine.